src/components/comment_list/comment_list.tsx

```diff
@@ -0,0 +1,64 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { HTMLAttributes, FunctionComponent } from 'react';
+import { CommonProps } from '../common';
+import classNames from 'classnames';
+import { EuiComment, EuiCommentProps } from './comment';
+
+export type EuiCommentListProps = HTMLAttributes<HTMLDivElement> &
+  CommonProps & {
+    /**
+     * List of comments to render. See #EuiComment
+     */
+    comments?: EuiCommentProps[];
+  };
+
+export const EuiCommentList: FunctionComponent<EuiCommentListProps> = ({
+  children,
+  className,
+  comments,
+  ...rest
+}) => {
+  const classes = classNames('euiCommentList', className);
+
+  let commentElements = null;
+
+  if (comments) {
+    commentElements = comments.map((item, index) => (
+      <EuiComment
```

Comment:
I just realized this is missing the possibility of adding an `...rest` items onto EuiComment. Like it doesn't take into account needing to pass `data-test-subj`. Since each `comments` item is a straight `EuiComment` type, you don't need to pass every single `item.prop` as you have it below. Instead, you should be able to just spread **all** props via `...item`. 

You'll still need a `key`, but this makes the code simpler and more flexible:
```suggestion
    commentElements = comments.map((item, index) => (
      <EuiComment
        key={index}
        {...item}
      />
```

Reply:
This is way cleaner, thanks! I made the change.